util/topsql/reporter/client.go

```diff
@@ -145,6 +148,7 @@ func (r *GRPCReportClient) sendBatchSQLMeta(ctx context.Context, sqlMap *sync.Ma
 	if err != nil {
 		return err
 	}
+	topSQLReportStatementHistogram.Observe(float64(cnt))
 	_, err = stream.CloseAndRecv()
```

Comment:
Should we also observe how long this operation takes?

Reply:
great, done.